Description: Re-introduce compat with gnutls__priority functions
 2.9.10 broke for programs using the old style gnutls_*_priority instead
 of gnutls_priority_* functions. 
Author: Nikos Mavrogiannopoulos <nmav@gnutls.org>
Origin: upstream
Bug-Debian: http://bugs.debian.org/579831

--- gnutls26-2.9.10.orig/lib/gnutls_state.c
+++ gnutls26-2.9.10/lib/gnutls_state.c
@@ -350,6 +350,11 @@ gnutls_init (gnutls_session_t * session,
 
   _gnutls_handshake_internal_state_init (*session);
 
+  /* emulate old gnutls behavior for old applications that do not use the priority_*
+   * functions.
+   */
+  (*session)->internals.priorities.unsafe_renegotiation = 1;
+
   return 0;
 }
 
